crates/service/src/builder.rs

```diff
@@ -106,7 +106,7 @@ impl<T: ServiceMsg, S: Service<Msg = T>> ServiceBuilder<S, TokioMpscInput<T>> {
     /// # Panics
     ///
     /// If an input has already been set.
-    pub fn create_command_handle(&mut self, capacity: usize) -> CommandHandle<S> {
+    pub fn create_command_handle(&mut self, capacity: usize) -> CommandHandle<T> {
```

Comment:
This is a very innocent but profound change.
Why do we need this diff?

Reply:
Otherwise we'd have to do something annoying like `CommandHandle<ChainWorkerService<StorageCtxImpl>>` and that makes any services using it dependent on how the chain worker service is instantiated by its direct consumer crate, or we'd have to leak the generic types all over the place.

I was trying to be clever and make it like how the `elliptic_curve` crate defines a type like `AffinePoint<C: Curve>`, which worked for the simple scenarios I was doing in my head, but it ended up not not working out since services aren't *entirely* self-contained.

Reply:
Ok, resolving